src/nvim/screen.c

```diff
@@ -1948,6 +1948,29 @@ static inline void provider_err_virt_text(linenr_T lnum, char *err)
   decor_add_ephemeral(lnum-1, 0, lnum-1, 0, &err_decor);
 }
 
+/// Draw the line number (empty space after wrapping).
+static inline void draw_line_number(win_T *wp, linenr_T lnum, char_u **extrap)
```

Comment:
Is `inline` really needed here?

Reply:
Not sure. 

I only did it because there's no need in creating a stack frame for this, as the change is only to improve readability. With the disclaimer that I'm a C novice, I generally add `inline` to most single use functions, especially when I'm hoisting code out in a refactor like this.

Is there a reason for not adding it? Or just let compiler figure this out.

Reply:
Oh, I trying compiling with `RelWithDebInfo` and the `inline` indeed seems to make a difference. This is fine then.

Reply:
Interesting...

What difference are you seeing?